pom.xml

```diff
@@ -4,12 +4,12 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.6.6</version>
+    <version>2.6.7</version>
     <relativePath /> <!-- lookup parent from repository -->
   </parent>
   <groupId>org.folio</groupId>
   <artifactId>folio-spring-base</artifactId>
-  <version>3.0.3-SNAPSHOT</version>
+  <version>3.0.4-SNAPSHOT</version>
```

Comment:
Please use a non-SNAPSHOT version.

Reply:
`mvn -DautoVersionSubmodules=true release:clean release:prepare` creates two commits:

[[maven-release-plugin] prepare release v3.0.3](https://github.com/folio-org/folio-spring-base/pull/51/commits/b23828e3338c045dccae316d36c373080a2c9ea8) [b23828e](https://github.com/folio-org/folio-spring-base/pull/51/commits/b23828e3338c045dccae316d36c373080a2c9ea8)
[[maven-release-plugin] prepare for next development iteration](https://github.com/folio-org/folio-spring-base/pull/51/commits/16f74088ec702541cc3141f190a13c043c9835b1) [16f7408](https://github.com/folio-org/folio-spring-base/pull/51/commits/16f74088ec702541cc3141f190a13c043c9835b1)

The first one changes the version to 3.0.3, this is used for the release tag.
The second one changes the version to 3.0.4-SNAPSHOT, this should have -SNAPSHOT because it is for the following development phase; it is NOT used for the release tag.

Reply:
Sure. My bad!